source/preferences.rst

```diff
@@ -440,7 +440,7 @@ Command
 
 
 ^^^^^^^
-Colours
+Colors
```

Comment:
Colours vs. Colors is a British vs American Spelling thing, and we've kind of settled on British spelling, as I'm Canadian and follow British spelling conventions, and all the previous authors were from the UK.

Reply:
I fixed it